tuxemon/core/states/world/worldstate.py

```diff
@@ -268,11 +268,11 @@ def broadcast_player_teleport_change(self):
             self.game.client.update_player(self.player1.facing)
 
         # Update the location of the npcs. Doesn't send network data.
-        for npc in self.npcs:
+        for slug,npc in self.npcs:
             char_dict = {"tile_pos": npc.tile_pos}
             networking.update_client(npc, char_dict, self.game)
 
-        for npc in self.npcs_off_map:
+        for slug,npc in self.npcs_off_map:
```

Comment:
Refer to my other comment [here](https://github.com/Tuxemon/Tuxemon/pull/203#discussion_r73624346) about this line.